platform/pc/platform.c

```diff
@@ -44,6 +44,9 @@ static bool found_acpi = false;
 
 /* multiboot information passed in, if present */
 extern uint32_t _multiboot_info;
+#if MULTIBOOT2_SUPPORT
+extern uint64_t _efi64_system_table;
```

Comment:
we should probably move these two a header, perhaps in hw/multiboot.h here and in the acpi_lite code.